test/unit/json_schema/draft4_invalid_schemas_test.dart

```diff
@@ -47,7 +47,7 @@ import 'package:test/test.dart';
 import 'package:json_schema/json_schema.dart';
 
 void main() {
-  final Directory testSuiteFolder = Directory('./test/invalid_schemas/draft4');
+  final Directory testSuiteFolder = Directory('./test/custom_tests/invalid_schemas/draft4');
```

Comment:
nit: `custom_tests` could probably just be called `custom`, since it lives within a directory already called `test` :man_shrugging:

Reply:
🤷  +1